did_core/did_doc/src/schema/legacy.rs

```diff
@@ -162,8 +164,9 @@ fn construct_new_did_document(
                 id,
                 did.clone(),
                 VerificationMethodType::Ed25519VerificationKey2018,
+                PublicKeyFormat::Multibase(multibase::Base::Base58Btc),
```

Comment:
Leaking the implementation details to the upper layer. Can we even reliably determine the key format at this point?